ublox_gps/config/rover.yaml

```diff
@@ -2,16 +2,16 @@
 ublox_gps_node:
   ros__parameters:
     debug: 1                    # Range 0-4 (0 means no debug statements will print)
-
+​
     save:
       mask: 3103                # Save I/O, Message, INF Message, Nav, Receiver
                                 # Manager, Antenna, and Logging Configuration
       device: 3                 # Save to FLASH and BBR
-
+​
     device: /dev/ttyACM0
     frame_id: gps
-    rate: 4.0                   # in Hz
-    nav_rate: 4                 # [# of measurement cycles], recommended 1 Hz, may
```

Comment:
Leave this parameters commented out, since they will be overwritten by the launch file